packages/base-zone/src/prepare-revocable.js

```diff
@@ -4,14 +4,22 @@ import { fromUniqueEntries } from '@endo/common/from-unique-entries.js';
 const { Fail, quote: q } = assert;
 
 /**
- * @typedef {object} Revoker
- * @property {() => boolean} revoke
+ * @template {any} [U=any]
```

Comment:
The `{any}` implied `U extends any`, which is a truism, since `any` is a top type.
```suggestion
 * @template [U=any]
```

Reply:
Done